lightly/api/api_workflow_upload_dataset.py

```diff
@@ -61,25 +61,25 @@ def upload_dataset(
         input: Union[str, "LightlyDataset"],
         max_workers: int = 8,
         mode: str = "thumbnails",
-        custom_metadata: Union[Dict, None] = None,
-    ):
-        """Uploads a dataset to to the Lightly cloud solution.
+        custom_metadata: Optional[Dict] = None,
+    ) -> None:
+        """Uploads a dataset to the Lightly Platform.
```

Comment:
Do you know if this endpoint still works? I don't think we support it anymore.

Reply:
Probably not. I didn't add an E2E test for this.